.gitignore

```diff
@@ -1,5 +1,4 @@
 *.class
-*.log
```

Comment:
would keeping all logs make sense?
If not maybe something like
```
*.log
!src/test/**/*.log
```
could make sense?

Reply:
good point, that works better.  :+1: